### PR TITLE
TMA-648 tests not deleting ads instances fixed

### PR DESCRIPTION
**Pull request by Hahihula**

### CODE REVIEW

File: spec/lcm/integration/m_to_n_spec.rb [hunk 1]
```diff
@@ -66,6 +66,13 @@
 
   after(:all) do
     @project.delete
+     begin
```
Comment (yenkins) [inline spec/lcm/integration/m_to_n_spec.rb]: Similar code found in m_to_n_spec.rb:69, visualisation_object_transfer_spec.rb:120, visualisation_object_transfer_spec.rb:127 (mass = 93)

Comment (yenkins) [inline spec/lcm/integration/m_to_n_spec.rb]: Style/IndentationConsistency: Inconsistent indentation detected.

File: spec/lcm/integration/visualisation_object_transfer_spec.rb [hunk 2]
```diff
@@ -114,4 +114,22 @@ def prepare_visualisation_object(rest_client, project)
     expect(prodVisualizationObject.first.meta['title']).to eq('Foo')
     expect(prodVisualizationObject.first.content['buckets'].first['items'].first['measure']['title']).to eq('Count of Account')
   end
+
+  after(:all) do
+    @project.delete
+    begin
```
Comment (yenkins) [inline spec/lcm/integration/visualisation_object_transfer_spec.rb]: Similar code found in m_to_n_spec.rb:69, visualisation_object_transfer_spec.rb:120, visualisation_object_transfer_spec.rb:127 (mass = 93)

File: spec/lcm/integration/visualisation_object_transfer_spec.rb [hunk 3]
```diff
@@ -114,4 +114,22 @@ def prepare_visualisation_object(rest_client, project)
...
+      GoodData.logger.info("Deleting ADS #{@ads.obj_id}")
+      @ads.delete
+    rescue StandardError => e
+      GoodData.logger.warn("Failed to delete development ADS. #{e}")
+      GoodData.logger.warn("Backtrace:\n#{e.backtrace.join("\n")}")
+    end
+    begin
```
Comment (yenkins) [inline spec/lcm/integration/visualisation_object_transfer_spec.rb]: Similar code found in m_to_n_spec.rb:69, visualisation_object_transfer_spec.rb:120, visualisation_object_transfer_spec.rb:127 (mass = 93)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/371/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1217/ : FAILURE in 1m 08s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1487/ : SUCCESS in 1m 06s

File: spec/lcm/integration/support/configuration_helper.rb [hunk 4]
```diff
@@ -47,6 +47,16 @@ def create_development_datawarehouse(opts = {})
       datawarehouse
     end
 
+    def delete_datawarehouse(ads)
+      begin
+        GoodData.logger.info("Deleting ADS #{ads.obj_id}")
```
Comment (yenkins) [inline spec/lcm/integration/support/configuration_helper.rb]: Calls 'gooddata.logger' 3 times (DuplicateMethodCall)

File: spec/lcm/integration/support/configuration_helper.rb [hunk 5]
```diff
@@ -47,6 +47,16 @@ def create_development_datawarehouse(opts = {})
...
+        ads.delete
+      rescue StandardError => e
```
Comment (yenkins) [inline spec/lcm/integration/support/configuration_helper.rb]: Has the variable name 'e' (UncommunicativeVariableName)

File: spec/lcm/integration/support/configuration_helper.rb [hunk 6]
```diff
@@ -47,6 +47,16 @@ def create_development_datawarehouse(opts = {})
       datawarehouse
     end
 
+    def delete_datawarehouse(ads)
+      begin
```
Comment (yenkins) [inline spec/lcm/integration/support/configuration_helper.rb]: Style/RedundantBegin: Redundant `begin` block detected.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/373/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1219/ : FAILURE in 1m 07s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1489/ : SUCCESS in 1m 10s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/403/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1248/ : FAILURE in 2m 37s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1518/ : SUCCESS in 2m 40s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/405/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1250/ : FAILURE in 1m 05s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1520/ : SUCCESS in 1m 01s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/408/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1253/ : FAILURE in 1m 07s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1523/ : SUCCESS in 1m 02s

**Comment by Hahihula** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/564/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1511/ : FAILURE in 24m 36s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/82/ : SUCCESS in 26m 10s

**Comment by Hahihula** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/565/ : SUCCESS in 33s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1512/ : FAILURE in 2m 03s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/83/ : SUCCESS in 33m 53s

**Comment by Hahihula** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/567/ : SUCCESS in 36s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1515/ : SUCCESS in 34m 53s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/84/ : SUCCESS in 29m 22s

**Comment by kubamahnert** [comment]
ok to merge

**Pull request merged**